### PR TITLE
add zone:ls command.

### PR DESCRIPTION
**Pull request by eyeamnoob**

### CODE REVIEW

**Comment by mhemrg** [comment]
Please rename the `dns` directory to `records`.

**Comment by mhemrg** [comment]
1. Please rename Hello to the corresponding class name on each file.
2. We use `I` at the beginning of each interface name not at the end. To be consistent with the rest of the project, please rename your interfaces as well

File: src/commands/zone/check.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import inquirer from 'inquirer';
+import Command, { IConfig } from '../../base.js';
+import { Flags } from '@oclif/core';
+import { createDebugLogger } from '../../utils/output.js';
+import { ux } from '@oclif/core';
+
+export default class Check extends Command {
+  static description = 'Check a zone.';
+
+  static baseURL = 'https://dns-service.iran.liara.ir';
```
Comment (mhemrg) [inline src/commands/zone/check.ts:10]: Please move this to the `constants.ts` file.

**Comment by mhemrg** [comment]
You can see here that column names are not consistent. Please use spaces for multi-word column names.
Can we also use colors for the Status column? Like the `db ls` command?

And also, the last checked at column needs to show the `hour:minute` as well.

Screenshot from 2023-09-25 14-30-37

File: src/commands/zone/check.ts (hunk 1 above)
Comment (eyeamnoob) [reply src/commands/zone/check.ts:10]: Right. We have other things that should be moved to its place like Interfaces. But for now, to keep avoiding merge conflict, I defined everything in a single file.

**Comment by eyeamnoob** [comment]
The module that we use to convert date, does not support time. Do i print the time in gregorian?

**Comment by mhemrg** [comment]
What about instead of showing the exact date and time, we show it in a "time ago" format? You can use the `moment` package to do this.

File: src/commands/zone/check.ts (hunk 1 above)
Comment (mhemrg) [reply src/commands/zone/check.ts:10]: OK. Don't forget to move them to the constants file after the merge.

**Pull request merged**